### PR TITLE
update(hardw softw requ): fix table & well syntax

### PR DESCRIPTION
**Pull request by NathalieC**
* Try to fix the Tables syntax
* Try to fix the well syntax

### CODE REVIEW

File: modules/ROOT/pages/hardware-and-software-requirements.adoc [hunk 1]
```diff
@@ -12,6 +12,7 @@ processes (number of processes instances, number of current users, operations an
 
 Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Portal)
 
+[width="30%"]
```
Comment (tbouffard) [inline modules/ROOT/pages/hardware-and-software-requirements.adoc]: `30%` seems too small

**With 30%**
image

**With default**
image

File: modules/ROOT/pages/hardware-and-software-requirements.adoc [hunk 2]
```diff
@@ -32,6 +33,7 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 
 Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Portal).
 
+[width="30%"]
```
Comment (tbouffard) [inline modules/ROOT/pages/hardware-and-software-requirements.adoc]: `30%` seems too small

**With 30%**
image


**With default**
image

**Review by tbouffard — CHANGES_REQUESTED** [review]
I have rebased the PR as the upstream branch had been reconverted
The proposed width for the tables seems too small. As there is no preview for the 7.5 branch, we can work together @NathalieC to find the best table width
We could also set a dedicated width for the first column.

File: modules/ROOT/pages/hardware-and-software-requirements.adoc (hunk 1 above)
Comment (NathalieC) [reply modules/ROOT/pages/hardware-and-software-requirements.adoc]: ok thanks. I think that 50% would be nice. I'm trying that.

File: modules/ROOT/pages/hardware-and-software-requirements.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/ROOT/pages/hardware-and-software-requirements.adoc]: again, trying 50%

**Comment by alachambre** [comment]
Why do you want to change the table width?? 
Playing with size like that can have a huge impact on mobile view, looks like a bad idea IMHO.

**Comment by alachambre** [comment]
I'm gonna close this pull request, no activity for a month, reopen it if needed.

**Pull request closed without merging**